packages/techdocs-cli/src/commands/index.ts

```diff
@@ -30,6 +30,12 @@ export function registerCommands(program: CommanderStatic) {
       "Output directory containing generated TechDocs site.",
       "./site/"
     )
+    .option(
+      "--docker-image <DOCKER_IMAGE>",
+      "The mkdocs docker container to use",
+      "spotify/techdocs"
+    )
+    .option("--no-pull", "Do not pull the latest docker image", false)
```

Comment:
A negating boolean option set by default `false` as the value, you can omit this third parameter...

Reply:
Ah, I didn't realize that was an option. Does omitting the value still show the default value in the help output?

Reply:
No it's not, even if you pass a default value, it's never shown in the helper...